osu.Game/Screens/Select/DifficultyRangeFilterControl.cs

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable disable
+using System;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Localisation;
+using osu.Game.Configuration;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Localisation;
+using osuTK;
+using osuTK.Graphics;
+
+namespace osu.Game.Screens.Select
+{
+    internal class DifficultyRangeFilterControl : CompositeDrawable
+    {
+        private Bindable<double> lowerStars;
+        private Bindable<double> upperStars;
+
+        private StarsSlider lowerSlider;
+        private MaximumStarsSlider upperSlider;
+
+        [BackgroundDependencyLoader]
+        private void load(OsuConfigManager config)
+        {
+            const float vertical_offset = 13;
+
+            InternalChildren = new[]
+            {
+                new OsuSpriteText
+                {
+                    Text = "Difficulty range",
+                    Font = OsuFont.GetFont(size: 14),
+                },
+                upperSlider = new MaximumStarsSlider
+                {
+                    Current = config.GetBindable<double>(OsuSetting.DisplayStarsMaximum),
+                    KeyboardStep = 0.1f,
+                    RelativeSizeAxes = Axes.X,
+                    Y = vertical_offset,
+                },
+                lowerSlider = new MinimumStarsSlider
+                {
+                    Current = config.GetBindable<double>(OsuSetting.DisplayStarsMinimum),
+                    KeyboardStep = 0.1f,
+                    RelativeSizeAxes = Axes.X,
+                    Y = vertical_offset,
+                },
+                upperSlider.Nub.CreateProxy(),
+                lowerSlider.Nub.CreateProxy(),
```

Comment:
I can't seem to find any change in behaviour when removing these nub proxies? It also makes sense in code because the upper slider only handles input on its right side, while the lower slider handles input on its left side (not to mention that the lower slider has transparent background).

Reply:
Have removed.